service/user/src/test/resources/application-documentation.yml

```
@@ -0,0 +1,54 @@
+      hibernate:
+        format_sql: true
+        show_sql: true
+        dialect: org.hibernate.dialect.H2Dialect
+    open-in-view: false
+
+  # Redis는 임베디드 Redis 사용
+  data:
+    redis:
+      host: localhost
+      port: 6370  # 임베디드 Redis 포트
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)